### PR TITLE
change enocdestring to encodebytes

### PR DESCRIPTION
**Pull request by SourceDoctor**
compatibility and upwards  for Python3.9

### CODE REVIEW

File: freerad_manager/django_freeradius/models.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import csv
-from base64 import encodestring
+from base64 import encodebytes as encodestring
```
Comment (peppelinux) [inline freerad_manager/django_freeradius/models.py:2]: Is this change retrocompatibile with py37 and py38?

Comment (peppelinux) [reply freerad_manager/django_freeradius/models.py:2]: If not

Protect the import with an if that checks the python version

Comment (SourceDoctor) [reply freerad_manager/django_freeradius/models.py:2]: ok which PYthon VErsion above you want me to use this

Comment (peppelinux) [reply freerad_manager/django_freeradius/models.py:2]: no worries, my bad, encodestyring was deprecated from 3.1 and 3.7 already has encodestring, so it is good as it is
thank you

**Pull request merged**